src/tools/impl/web.py

```
@@ -21,26 +20,22 @@ def _extract_host_from_url(url: str) -> str:
         return url
 
 
-def _render_url_args(args: dict, config: RunnableConfig) -> str:
+def _render_url_args(args: dict, config: dict) -> str:
```

Ruff (0.14.3)

Unused function argument: `config`

(ARG001)